.github/workflows/ci.yml

```diff
@@ -15,19 +15,12 @@ jobs:
       fail-fast: false
       matrix:
         java-version:
-          - 1.8
-          - 9
-          - 10
+          - 8
```

Comment:
Do you know what the difference is here between `8` and `1.8` as far as this change is concerned?

Reply:
The new version of the action does not support the old term of `1.8`, they want `8` now. Same thing though.